includes/class-events-store.php

```diff
@@ -147,10 +148,10 @@ protected function _prepare_table() {
 		dbDelta( $schema, true );
 
 		// Confirm that the table was created, and set the option to prevent further updates.
-		$table_count = count( $wpdb->get_col( $wpdb->prepare( 'SHOW TABLES LIKE %s', $table_name ) ) );
+		$is_installed = 1 === count( $wpdb->get_col( $wpdb->prepare( 'SELECT TABLE_NAME FROM information_schema.tables WHERE TABLE_NAME = %s', $table_name ) ) );
```

Comment:
This makes me wonder if we really need this query in the first place. For `is_installed` to ever be false, `CREATE TABLE` must have failed somehow. Can't we simply depend on `dbDelta` successfully executing the query? Otherwise, are we assuming that we can't rely on `dbDelta` to work correctly (or to handle failures?)

Reply:
@WPprodigy based on our conversation, I did some debugging around the `$use_master` flag in `db_connect`. In this case, it does set it to use the master db. (This is during site creation, right after creating the table.)

<img width="1691" alt="image" src="https://github.com/Automattic/Cron-Control/assets/6265975/5ee8d2e1-bc75-4217-8cdf-06381b374cf2">

in the screenshot:
1. The query is our new information schema query
2. It's set to use master
3. The new cron_control_jobs table is listed in srtm.

Reply:
perfect! thanks for checking